### PR TITLE
week1

### PR DESCRIPTION
**Pull request by xajaxx**
### What test did you complete before submitting the PR?

### CODE REVIEW

File: Week1-Database-Introduction/DongChuangshi-Week1/董创轼实验报告.md [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+gremlin> g.V(pluto).out('brother').as('god').out('lives').as('place').select('god', 'place')
+==>[god:v[4280],place:v[4344]]
+==>[god:v[4328],place:v[4216]]
+//都替换成名字
+gremlin> g.V(pluto).out('brother').as('god').out('lives').as('place').select('god', 'place').by('name')
+==>[god:jupiter,place:sky]
+==>[god:neptune,place:sea]
+```
+
+8、
```
Comment (ehds) [inline Week1-Database-Introduction/DongChuangshi-Week1/董创轼实验报告.md:405]: 增加 8 的说明

File: Week1-Database-Introduction/DongChuangshi-Week1/董创轼实验报告.md [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+
+解决方法为，将文件
+
+*/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/accessibility.properties*
+
+注释掉 
+
+#assistive_technologies=org.GNOME.Accessibility.AtkWrapper
+
+3、
```
Comment (ehds) [inline Week1-Database-Introduction/DongChuangshi-Week1/董创轼实验报告.md:435]: 增加 3 后的说明 `加载图出错`

**Review by karmanluo — COMMENTED** [review]
Line 305, Need change local picture path to network path.

**Review by ehds — CHANGES_REQUESTED** [review]

**Review by ehds — APPROVED** [review]

**Comment by ehds** [comment]
> Line 305, Need change local picture path to network path.

你在review的时候可以直接标记位置。

File: Week1-Database-Introduction/DongChuangshi-Week1/董创轼实验报告.md [hunk 3]
```diff
@@ -0,0 +1,452 @@
...
+#加载图
+gremlin> graph = JanusGraphFactory.open('conf/janusgraph-inmemory.properties')
+==>standardjanusgraph[inmemory:[127.0.0.1]]
+gremlin> GraphOfTheGodsFactory.loadWithoutMixedIndex(graph, true)
+==>null
+#获取诸神关系实例
+gremlin> g = graph.traversal()
+==>graphtraversalsource[standardjanusgraph[inmemory:[127.0.0.1]], standard]
+```
+[graph-of-the-gods](https://github.com/dongcss/training-plan/blob/master/images/graph-of-the-gods-2.png)
```
Comment (ehds) [inline Week1-Database-Introduction/DongChuangshi-Week1/董创轼实验报告.md:305]: add `!` before `[graph-of-the-gods]`.

**Review by karmanluo — APPROVED** [review]
LGTM

**Pull request closed without merging**